tagcompare/compare.json

```diff
@@ -16,6 +16,9 @@
         "browserName": "chrome"
       }
     },
+    "phantomjs": {
+      "enabled": true
+    },
```

Comment:
I was thinking mostly the whole file. Reading it, I'm not entirely sure what it does, or why it's needed. Usually when I see config files, I think about why I'd want to edit them. Like, I want to edit /etc/apache.conf because I don't want to recompile Apache to change my website configuration.

Maybe this file has a good use, but I can't tell and that's also a problem. If you want to keep it, what do you think about using YAML so you can at least have comments?